src/cli/main.go

```diff
@@ -11,25 +11,33 @@ import (
 )
 
 func main() {
-	fromXML, err := parsers.ParseAlphabet("parsers/xmlittre-data")
-	if err != nil {
-		fmt.Println(err)
-		return
+	var parsedEntries []dictionary.Entry
+	var err error
+
+	if _, err = os.Stat("bin/dict.gob"); os.IsNotExist(err) {
```

Comment:
defer the knowledge of the gob path to the gob lib

![goblin](https://user-images.githubusercontent.com/11455210/119276508-e4479480-bc1a-11eb-92b3-f9b66e502eb4.gif)

Reply:
Quite the opposite. The lib should not decide where it writes, if you want it to be reusable.